src/test/java/org/zalando/nakadi/controller/SubscriptionControllerTest.java

```diff
@@ -250,11 +265,43 @@ public void whenPostSubscriptionAndExceptionThenServiceUnavailable() throws Exce
 
     private ResultActions getSubscriptions(final Optional<String> owningApplication) throws Exception {
         final String url = "/subscriptions" + owningApplication.map(app -> "?owning_application=" + app).orElse("");
-        return mockMvc.perform(get(url));
+        return mockMvcBuilder.build().perform(get(url));
+    }
+
+    @Test
+    public void whenPostSubscriptionWithNoReadScopeThenForbidden() throws Exception {
+        when(eventTypeRepository.findByName("myET")).thenReturn(EventTypeTestBuilder.builder()
+                .name("myET")
+                .readScopes(Collections.singleton("oauth.read.scope"))
+                .build());
+
+        final SubscriptionBase subscriptionBase = randomSubscription()
+                .withOwningApplication("app")
+                .withEventTypes(ImmutableSet.of("myET"))
+                .buildSubscriptionBase();
+        postSubscription(subscriptionBase)
+                .andExpect(status().isForbidden())
+                .andExpect(content().string(new StringContains("Client has to have scopes: [oauth.read.scope]")));
```

Comment:
But why you don't just check for problem here? Like here 
https://github.com/zalando/nakadi/pull/373/files#diff-8cb6844b0c2bfa55bfd6a5e918d7e925R263
It will also check that the response structure is a problem object